tripal_chado/includes/TripalImporter/GFF3Importer.inc

```diff
@@ -2821,6 +2893,7 @@ class GFF3Importer extends TripalImporter {
     $name = '';
 
     // If there is no ID or name then try to create a name and ID.
+    // SOFIA: Name typo?? should name be Name?
     if (!array_key_exists('ID', $attrs) and !array_key_exists('name', $attrs)) {
```

Comment:
Good catch! Yes, it should be Name (capitalized).

Reply:
Changed 'name' to 'Name'.